bin/Powheg/production/pre2017/13TeV/DY_MiNNLO_NNPDF31_13TeV/minnloHelper.sh

```diff
@@ -27,10 +27,10 @@ case $WHAT in
     ;;
     
     INIT )
-        ln -s genproductions/bin/Powheg/*.py .
-        ln -s genproductions/bin/Powheg/*.sh .
-        ln -s genproductions/bin/Powheg/patches .
-        ln -s genproductions/bin/Powheg/production/pre2017/13TeV/DY_MiNNLO_NNPDF31_13TeV .
+        ln -s ../../genproductions/bin/Powheg/*.py .
+        ln -s ../../genproductions/bin/Powheg/*.sh .
+        ln -s ../../genproductions/bin/Powheg/patches .
+        ln -s ../../genproductions/bin/Powheg/production/pre2017/13TeV/DY_MiNNLO_NNPDF31_13TeV .
```

Comment:
Could you put that in a new option like `INIT2`, please? I personally prefer my genproductions repository to stay clean of generator code 😅

Reply:
Ah, I see, good deal! I was usually (sparse) checking out genproductions within each CMSSW, let's use your strategy then 👍